sdk/storage/azure-storage-blob/azure/storage/blob/_shared/avro/avro_io.py

```diff
@@ -0,0 +1,616 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""Input/output utilities.
```

Comment:
I know at least in Java and .NET, they wanted us to add the word internal into the namespace so its clear its internal. Not sure if that applies to Python

Reply:
the `_` in `_shared` says it's private.

See https://stackoverflow.com/questions/1301346/what-is-the-meaning-of-a-single-and-a-double-underscore-before-an-object-name